packages/ember-glimmer/lib/syntax/let.ts

```diff
@@ -1,37 +1,37 @@
-import { compileList } from "@glimmer/runtime";
-
 /**
 @module ember
+*/
 
-/**
```

Comment:
Tacking a `?w=` to the URL makes this diff much more legible. https://github.com/emberjs/ember.js/pull/16112/files?w=1